4.4/4.4.8/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 4.4.8
+Создать объект, внутри которого будет изначально пустой объект, а дальше будем его наполнять новыми пустыми свойствами
+со
+следующей логикой:
+Появляется сообщение: "Введите название нового свойства"
+Если юзер вводит название и нажимает ОК - добавляется новое свойство к внутреннему новому отдельному объекту и
+потом снова появляется сообщение с предложением ввести название свойства.
+Если нажимает ОТМЕНА - выводится только алерт с перечнем всех свойств (которые добавлял юзер) этого отдельного
+внутреннего объекта, таким образом: "{ prop1, prop2, prop3, ...., }".
+<script>
+	const obj = {
+		newUserObj: {},
+		getNewProperty() {
+			let isContinue = false;
```

Comment:
it's a bit strange - we are starting the cycle with the `isContinue` flag which is false